todo_app/data/db_items.py

```diff
@@ -0,0 +1,55 @@
+import pymongo
+import os
+
+class Item:
+    def __init__(self, _id, name, status = 'To Do'):
+        self._id = _id
```

Comment:
I'd argue this should stay as `id` to the outside world, e.g.:
```suggestion
        self.id = _id
```

That way the rest of our code doesn't need to care that the underlying code has changed to Mongo, and we don't need to make changes like the one in `index.html`